File: lang/python/src/fun/vac/util/TestRunner.py
import sys

global TestRunner_TestIndex
TestRunner_TestIndex = 0

def parseTest(ok: bool) -> None:
    global TestRunner_TestIndex

    if ok:
        print("< Test [%d] Passed >" % TestRunner_TestIndex)
    else:
        print("X Test [%d] Failed X" % TestRunner_TestIndex, file = sys.stderr)

    TestRunner_TestIndex += 1

File: lang/go/src/fun/vac/util/TestRunner/TestRunner.go
package TestRunner

import (
    "fmt"
    "os"
)

var TestRunner_TestIndex int

func ParseTest(ok bool) {
    if ok {
        fmt.Printf("✓ Test [%d] Passed\n", TestRunner_TestIndex)
    } else {
        fmt.Fprintf(os.Stderr, "✗ Test [%d] Failed\n", TestRunner_TestIndex)
    }

    TestRunner_TestIndex += 1
}
